packages/machine/src/virtual-app-key.ts

```diff
@@ -0,0 +1,11 @@
+import { keccak256 } from "ethers/utils/solidity";
+
+export function virtualAppKey(users: string[], intermediary: string) {
```

Comment:
Can you explain the usage / requirement of this?

Reply:
as discussed, one place to put virtual app target app instances is a separate state channel. this function computes the key for that channel.

other possible approaches:
- have `virtualAppKey` be the `create2` address of a multisig with `users + [intermediary]` as owners. I do not like this approach because it means that this function changes if we upgrade the multisig code. furthermore, it is confusing because we explicitly never deploy a multisig currently. we already have a field on statechannel for the owner that can be set to the `create2` address in the future if needed
- change things so that channels are not forced to be keyed by a multisig address. I think this is the right approach, however it involves more work, for e.g., changing the store